docker/dockerfiles/fe/fe-ubuntu.Dockerfile

```diff
@@ -53,5 +56,4 @@ COPY --chown=starrocks:starrocks docker/dockerfiles/fe/*.sh $STARROCKS_ROOT/
 # Create directory for FE metadata
 RUN mkdir -p /opt/starrocks/fe/meta
 
-# run as root by default
-USER root
+USER $RUN_USER
```

Comment:
line 43, 44, 48, shall keep the same as this RUN_USER, otherwise the user/group permission will be not the one as `RUN_USER` and will face the permission issue accessing the directories and files. similar issue to be-ubuntu image.

Reply:
good point!